Makefile

```diff
@@ -102,7 +107,10 @@ extractor/docker:
 extractor/install:
 	@cd extractor && npm run build
 
-extractor/test:
+extractor/node_modules: extractor/package-lock.json
```

Comment:
I think there's a line earlier in this file that does this same cd && npm install .. that target could depend on this one.

Reply:
I see https://github.com/depscloud/depscloud/blob/main/Makefile#L17 ... It probably should, and this should probably be using install instead of ci. I'll update.